### DISCUSSION TITLE
post/frontend-study-journey/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#40] [Announcements]
# post/frontend-study-journey/

카카오페이 입사 후 3번의 스터디를 진행한 방식과 그 과정들을 설명합니다. 주니어 개발자들이 스터디를 진행하면서 느낀 좋았던 점과 아쉬웠던 점, 그리고 나름의 스터디 꿀팁들을 공유합니다.

https://tech.kakaopay.com/post/frontend-study-journey/

**Comment by yrnana** [comment]
페이 FE 최고!!
